source/frontend/src/components/common/HeaderField/AuditSection.tsx

```diff
@@ -51,3 +51,10 @@ export const StyledSmallText = styled.span`
   font-size: 0.87em;
   line-height: 1.9;
 `;
+
+const StyledCol = styled(Col)`
+  text-align: right;
```

Comment:
draw these fields on the left when the screen is too small to display them on the same row consistently:

![image](https://github.com/user-attachments/assets/d505733a-57d3-4a70-b048-1e043af7f0c1)